easylistportuguese/easylistportuguese_specific_hide.txt

```diff
@@ -244,4 +244,5 @@ brasil247.com##.adBackground
 milkpoint.com.br##a[class^="lnkbn"]
 publishnews.com.br##div[class^="pn-anuncio"]
 fogaonet.com##div[class*="fnad-block"]
-vagalume.com.br###cazamba_vagalume_passback_container
\ No newline at end of file
+vagalume.com.br###cazamba_vagalume_passback_container
+maisesports.com.br##a[href^="https://mais.gg/cyberbet"]
```

Comment:
After adding the blocking filter, is this really needed? I didn't see any empty placeholder after that. Also, the url on the href changes depending on the ad, so your filter is not always working. If you think it would be needed after adding the blocking rule I would consider something more generic.
`maisesports.com.br##a[href^="https://mais.gg/"]`